dns/dnscrypt-proxy/src/www/diag_logs_dnscrypt.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$type = 'dnscrypt-proxy';
+if (isset($_GET['type']) && ($_GET['type'] === 'dnscrypt-proxy' || $_GET['type'] === 'query' || $_GET['type'] === 'nx')) {
```

Comment:
You may use in_array in I remember that correctly or use a switch block.

Reply:
I just used the style from bind which was fixed by @fichtner so I think this one is ok.